csrankings-j.csv

```diff
@@ -136,6 +136,7 @@ Jaelson Freire Brelaz de Castro,UFPE,http://www.cin.ufpe.br/~jbc,Um1B6_4AAAAJ
 Jaesik Choi,KAIST,http://sailab.kaist.ac.kr/jaesik,RqMLVzUAAAAJ
 Jaesik Park,POSTECH,http://jaesik.info,_3q6KBIAAAAJ
 Jaewoo Kang,Korea University,http://dmis.korea.ac.kr/people,iiIxp8cAAAAJ
+Jaewoong Sim,Seoul National University,https://jaewoong.org,4U65ziwAAAAJ

```

Comment:
ECE, not CS. Please explain how they meet the inclusion criteria (publishing in CS venues is _not_ sufficient). Thanks.

Reply:
Hi Emery. Thanks for reviewing my request.

I do computer architecture and systems work (publishing at e.g., ISCA/MICRO/ASPLOS/HPCA) in the Computer Track, and I can solely advise PhD students if they are enrolled in the College of Engineering (including CSE, which is CS at SNU); that is, a CS PhD student can select me as a thesis advisor. This is a university-level policy, and the policy document seems to be only written in Korean (according to the conversation with the school this morning). I can still send it to you if that helps. Thanks!

Reply:
That's fine, thanks.